storage-proofs/porep/src/stacked/vanilla/proof.rs

```diff
@@ -973,34 +973,31 @@ impl<'a, Tree: 'static + MerkleTreeTrait, G: 'static + Hasher> StackedDrg<'a, Tr
             CacheKey::CommDTree.to_string(),
             Some(get_merkle_tree_len(nodes_count, BINARY_ARITY)?),
         );
-        tree_d_config.rows_to_discard =
-            StoreConfig::default_rows_to_discard(nodes_count, BINARY_ARITY);
+        tree_d_config.rows_to_discard = default_rows_to_discard(nodes_count, BINARY_ARITY);
 
         let mut tree_r_last_config = StoreConfig::from_config(
             &config,
             CacheKey::CommRLastTree.to_string(),
             Some(get_merkle_tree_len(nodes_count, Tree::Arity::to_usize())?),
         );
 
-        // A default 'rows_to_discard' value will be chosen for
-        // tree_r_last, unless the user overrides this value via the
-        // environment setting (FIL_PROOFS_ROWS_TO_DISCARD).  If this
-        // value is specified, no checking is done on it and it may
+        // A default 'rows_to_discard' value will be chosen for tree_r_last, unless the user overrides this value via the
+        // environment setting (FIL_PROOFS_OCT_ROWS_TO_DISCARD).  If this value is specified, no checking is done on it and it may
```

Comment:
It's still not clear to me whether this is converging on what I think the right answer should be:

- If nothing is specified by 'settings', default to discarding zero rows.
- If a setting is specified by the user (env var or config file), use that.
- If nothing is specified by the user, use a default hard-coded in settings.

Reply:
This doesn't address the first case at all, as something is always specified by the settings (either the default value, or a user provided value).  The last 2 cases should be satisfied.